scss/_patterns_navigation.scss

```diff
@@ -12,6 +13,11 @@ $spv-navigation-logo: 0.3125rem;
   %navigation-link-responsive-padding-vertical {
     padding-bottom: $spv--large;
     padding-top: $spv--large;
+
+    @media (max-width: $breakpoint-navigation-threshold) {
```

Comment:
Ideally we should do this other way around (mobile first), so by default smaller spacing, and `min-width` breakpoint to turn it into desktop view.